src/main/java/ip/task/Event.java

```diff
@@ -0,0 +1,62 @@
+package ip.task;
+
+/**
+ * A subclass of the Task class. Contains the additional information
```

Comment:
Redundant info when you say `subclass of the Task ...`; it is already evident from `extends` in the code 🙂